<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriveForm</class>
 <widget class="QWidget" name="DriveForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Axis Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="axisName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Op Mode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="opMode">
     <item>
      <property name="text">
       <string>Position</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Velocity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Torque</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Actual Position</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="actualPos">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Actual Velocity</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="actualVel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Actual Torque</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="actualTor">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Load Torque</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="loadTor">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Axis Status</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="axisStatus">
     <property name="text">
      <string>unknown</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Motion Status</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="motionStatus">
     <property name="text">
      <string>unknown</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>PTP Absolute</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="ptpAbs">
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>PTP Relative</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLineEdit" name="ptpRel">
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QPushButton" name="parameters">
     <property name="text">
      <string>Parameters</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
